### PR TITLE
[#OSF-7628]Add Physical COS Address to Transactional\Notification Emails

### PR DESCRIPTION
**Pull request by chennan47**
## Purpose
Add Physical COS Address to Transactional\Notification Emails

## Changes
All the emails sent by OSF should now contain a physical address of COS which is
Center for Open Science
210 Ridge McIntire Road
Suite 500
Charlottesville, VA 22903-5083

and a link to our privacy policy which is https://github.com/CenterForOpenScience/cos.io/blob/master/PRIVACY_POLICY.md

## QA notes:
https://docs.google.com/document/d/1n4uLnbiJkOLqj3wIbUnd0Cuj03UkG0AXg-q-eowxtyo/

## Ticket

https://openscience.atlassian.net/browse/OSF-7628

### CODE REVIEW

File: website/templates/emails/comment_replies.txt.mako [hunk 1]
```diff
@@ -1,3 +1,10 @@
...
-${'\t'}To view this on the Open Science Framework, please visit: ${url}.
\ No newline at end of file
+${'\t'}To view this on the Open Science Framework, please visit: ${url}.
+
+
+Center for Open Science
+210 Ridge McIntire Road
+Suite 500
+Charlottesville, VA 22903-5083
+Privacy Policy: https://github.com/CenterForOpenScience/cos.io/blob/master/PRIVACY_POLICY.md
```
Comment (alexschiller) [inline website/templates/emails/comment_replies.txt.mako]: Driveby comment: 
The address/privacy policy is exactly the same in most of these (aside from some with formatting). 
Is there a way this could be made into a variable and put into something like `language.py` in case other changes to the signature are required (e.g., we move or change the location of the Privacy Policy)? 

Just a idea, might be worth looking into.

File: website/templates/emails/file_updated.html.mako [hunk 2]
```diff
@@ -7,6 +7,13 @@
                 <span class="person" style="font-weight: bold;">${user.fullname} </span>
                 ${message}
             </span>
+            <tr style="border-collapse: collapse;" align="left">
+                <br>
+                Center for Open Science<br>
+                210 Ridge McIntire Road<br>
+                Suite 500<br>
+                Charlottesville, VA 22903-5083<br>
```
Comment (alexschiller) [inline website/templates/emails/file_updated.html.mako]: No privacy policy on this one?

File: website/templates/emails/mentions.html.mako [hunk 3]
```diff
@@ -10,6 +10,13 @@
             <span class="timestamp" style="color: grey;"> at ${localized_timestamp}: </span>
             <span class="content" style="display: block;padding: 6px 5px 0px 8px;font-size: 14px;">${content}</span>
         </td>
+        <tr style="border-collapse: collapse;" align="left">
+            <br>
+            Center for Open Science<br>
+            210 Ridge McIntire Road<br>
+            Suite 500<br>
+            Charlottesville, VA 22903-5083<br>
```
Comment (alexschiller) [inline website/templates/emails/mentions.html.mako]: privacy policy?

File: website/templates/emails/prereg_challenge_accepted.html.mako [hunk 4]
```diff
@@ -22,3 +22,8 @@ Thank you for entering the Preregistration Challenge. Feel free to submit anothe
 Sincerely,
 <br />
 The team at the Center for Open Science
+<br>
+Center for Open Science<br>
+210 Ridge McIntire Road<br>
+Suite 500<br>
+Charlottesville, VA 22903-5083<br>
```
Comment (alexschiller) [inline website/templates/emails/prereg_challenge_accepted.html.mako]: privacy policy

File: website/templates/emails/comment_replies.txt.mako (hunk 1 above)
Comment (brianjgeiger) [inline website/templates/emails/comment_replies.txt.mako]: Ugh. Yeah, we need to have actual templates for this. I believe there's a ticket to take care of this at some point.
